python/opentracing/client.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+#
+
+from os import getenv
+from random import randint
+from time import sleep
+
+from requests import get
+
+# from opentracing import set_global_tracer, global_tracer
+from opentracing import set_global_tracer
+from lightstep import Tracer
+
+# Added opentelemetry lines
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.instrumentation.opentracing_shim import create_tracer
+from opentelemetry.launcher import configure_opentelemetry
+
+trace.set_tracer_provider(TracerProvider())
+configure_opentelemetry()
+otel_tracer = trace.get_tracer(__name__)
+shim = create_tracer(trace.get_tracer_provider())
+
+
+def send_requests(target):
+    integrations = ["pymongo", "redis", "sqlalchemy"]
+    # with global_tracer().start_active_span("client operation"):
+    with shim.start_active_span("client operation"):
```

Comment:
is it possible to use the global_tracer here?

Reply:
Fixed